Fixes to README.es_ES.md

gubatron opened the pull request:
- Fixed one typo
- Months in Spanish are not spelled with a first capital letter by convention, it's weird, but it's the way it is
- Italicized some terms that remained in English
- One grammar fix (s/Por cualquier/Para cualquier/)

gubatron committed:
Fixes to README.es_ES.md


- Fixed one typo
- Months in Spanish are not spelled with a first capital letter by convention, it's weird, but it's the way it is
- Italicized some terms that remained in English
- One grammar fix (s/Por cualquier/Para cualquier/)

platonbjs commented on locales/README.es_ES.md:
```diff
@@ -18,13 +18,13 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
 > **P:** ¿A qué te refieres con, "Centos ha cambiado de dirección"?
 
-El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualziaciones *antes* de su inclusión en el prove ...
+El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualizaciones *antes* de su inclusión en el prove ...
```
When it says "recibía parches y actualizaciones luego del proveedor", maybe it's better "recibía parches y actualizaciones después del proveedor"

bezunartea replied:
Sorry for the delay! Yes, I agree that "después" is better than "luego".

platonbjs commented on locales/README.es_ES.md:
```diff
@@ -18,13 +18,13 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
+El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualizaciones *antes* de su inclusión en el prove ...
 
-En adición a eso, el soporte para CentOS Linux 8 se acortó, del 31 de Mayo de 2029 al 31 de Diciembre de 2021.
+En adición a eso, el soporte para CentOS Linux 8 se acortó, del 31 de mayo de 2029 al 31 de diciembre de 2021.
```
Maybe better "Añadido a eso" or "Además de eso"

gubatron replied:
quizás esto seria bueno para otro parche.
(solo arreglando errores de convenciones con respecto a fechas en este parrafo)

Una cosa a la vez.

bezunartea replied:
"Añadido a eso" or "Además de eso" is much better than "En adición a eso"... "Adicionalmente" may be the best?

platonbjs commented on locales/README.es_ES.md:
```diff
@@ -18,13 +18,13 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
 > **P:** ¿Y dónde entra Rocky Linux?
 
-Rocky Linux apunta a funcionar como un build downstream como lo hacía previamente CentOS, compilando releases después de que se agreguen al proveedor upstream, no antes.
+Rocky Linux apunta a funcionar como un build *downstream* como lo hacía previamente CentOS, compilando releases después de que se agreguen al proveedor *upstream*, no antes.
```
Maybe better "Rocky Linux aspira a ser una build..."

gubatron replied:
hmm, para mi es "un build" en masculino. Aunque pudiera ser una build porque es una distribución.
Ok.

gubatron committed:
Update README.es_ES.md

s/un build/una build/g

Se considera en femenino por ser un build de una distribución Linux.
